website/src/utils/animation-utils.js

```diff
@@ -1,20 +0,0 @@
-/* global window */
```

Comment:
This file is used here too: https://github.com/uber/deck.gl/blob/c534253cdad3cc26abf358d3b5017c4e28a3de9b/website/src/components/home.js#L8

Reply:
Removed in this PR

Reply:
oh yes, all good, thanks!